pypesto/objective/amici_objective.py

```diff
@@ -289,29 +277,35 @@ def _call_amici(
             num_threads=min(self.n_threads, len(self.edatas)),
         )
 
+        par_sim_ids = list(self.amici_model.getParameterIds())
+
         for data_ix, rdata in enumerate(rdatas):
             log_simulation(data_ix, rdata)
 
             # check if the computation failed
             if rdata['status'] < 0.0:
                 return self.get_error_output(rdatas)
 
+            condition_map_sim_var = self.parameter_mapping[data_ix][2]
```

Comment:
Replace `2` by something descriptive?

Reply:
that's just the index ... I know, a 6-tuple is not ideal to have. maybe we should create something more meaningful from the mapping object, make a class, ...?

Reply:
Some `IDX_PAR_MAP_SIM_VAR=...` would already do the job for me.

Reply:
sounds good